src/encoded/static/components/experiment_series.js

```diff
@@ -520,10 +522,16 @@ class ExperimentSeriesComponent extends React.Component {
         if (Object.keys(viewableDatasets).length > 0) {
             // Add the "Add all to cart" button and internal tags from all related datasets.
             const experimentIds = Object.values(viewableDatasets).map(experiment => experiment['@id']);
+            const hubUrl = `http://genome.ucsc.edu/cgi-bin/hgTracks?hubClear=${window.location.origin}/batch_hub/type=Experiment%26accession=${
+                Object.values(viewableDatasets).map(experiment => experiment.accession).join('%26accession=')
```

Comment:
I would just use question marks instead of %26 to make this easier to understand. Using question marks won’t cause a 301 redirect — our server just accepts it.

Reply:
That's `&`. I put it as %26 to not confuse UCSC. I tested `&` and it won't work.